doc/rtd/topics/faq.rst

```diff
@@ -121,6 +121,12 @@ cloud-init:
   $ sudo cloud-init init --local
   $ sudo cloud-init init
 
+.. warning::
```

Comment:
+1 but maybe worth trying to format to stand out a bit more. In place in the doc it doesn't have that "WARNING-ness" to it.

Reply:
Not 100% sure I follow this: the warning has a distinct orange background and takes up about a third of this answer's vertical space in the rendered docs (which I screenshotted in Test Steps, for reference). Could you suggest more specifically how you'd like to see this improved?

Reply:
Apologies, I was looking at it in the GH render which of course makes no sense because it's going to be parsed by RTD and is rst vs md.